### ISSUE TITLE
(draft) Module `defusedxml.lxml` will be deprecated

### ISSUE THREAD

**Issue by glarrain** [fyndata/lib-cl-sii-python#31]
We rely on function `defusedxml.lxml.fromstring()` for our function `libs.xml_utils.parse_untrusted_xml()`. What should we use instead?

> defusedxml.lxml
> DEPRECATED The module is deprecated and will be removed in a future release.

[Source](https://github.com/tiran/defusedxml/blob/v0.6.0/README.md#defusedxmllxml)

CC @jtrh

**Comment by jtrh** [comment]
The owner of the [defusedxml repository](https://github.com/tiran/defusedxml) is thinking of ways to solve this issue (the deprecation part, at least): https://github.com/tiran/defusedxml/issues/38#issuecomment-488647815.

Is the function `defusedxml.lxml.fromstring()` they only part of `defusedxml.lxml` that we use?

**Comment by glarrain** [comment]
> Is the function defusedxml.lxml.fromstring() they only part of defusedxml.lxml that we use?

Yes. However, since that uses lxml, the XML-related classes that we reference all over the place in that project and others are aliases to lxml classes:

https://github.com/fyndata/lib-cl-sii-python/blob/cde9fe6e09a4e97bee55134c7419eb62fecd5111/cl_sii/libs/xml_utils.py#L31-L34

**Comment by jtrh** [comment]
SignXML removed the DefusedXML dependency by incorporating its useful aspects. See https://github.com/XML-Security/signxml/commit/83c05fbfdc9efbea0ddc13cd490b423bb772ef5d.

**Comment by glarrain** [comment]
Useful stuff:
- https://github.com/XML-Security/signxml/blob/master/signxml/util/__init__.py
- https://github.com/XML-Security/signxml/blob/83c05fbfdc9efbea0ddc13cd490b423bb772ef5d/signxml/__init__.py#L185-L245
